backend/audit/test_schemas.py

```diff
@@ -872,7 +865,10 @@ def test_number_of_audit_findings(self):
             simple_case["FederalAwards"]["federal_awards"][0]["program"][
                 "number_of_audit_findings"
             ] = 0
-            self.assertRaises(exceptions.ValidationError, validate, simple_case, schema)
+            # We cannot find, in the UG, anything that suggests this is true.
+            # This seems like it should be allowed to pass.
+            # self.assertRaises(exceptions.ValidationError, validate, simple_case, schema)
+            validate(simple_case, schema)
```

Comment:
Since this change reverses the meaning of the `test_number_of_audit_findings` test, would it be better to omit it altogether until we're clear on the requirements from UG?

Reply:
Makes sense. On the other hand, because we are now allowing 0 through the schema, the test case should reflect what we have in the schema and when we update the schema, this test will fail as a reminder for us to update the test cases accordingly.